### PR TITLE
update restControllerEntityJdbc

### PR DESCRIPTION
**Pull request by AndriyHishchak**
Updated the return of all methods of the ResponseEntity object. Also updated the getById method and fixed endpoints.

### CODE REVIEW

File: src/main/java/com/project/Investment/App/controller/jdbc/RestControllerEntityJdbc.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    ) {
+        return  ResponseEntity.status(HttpStatus.OK).body(service.findById(id, effectiveDate, limit)));
+    }
+    @GetMapping("/benchmarkId/{id}")
+    public ResponseEntity<List<Entity>> getByDefaultBenchmarkId(@PathVariable(name = "id") String id) {
+        return  ResponseEntity.status(HttpStatus.OK).body(service.findByDefaultBenchmarkId(id));
+    }
+
+    @GetMapping()
+    public ResponseEntity<List<Entity>> getAll(@RequestParam(value = "name", required = false) Optional<String> name) {
```
Comment (romanrudyidev) [inline src/main/java/com/project/Investment/App/controller/jdbc/RestControllerEntityJdbc.java]: You should either use `name` in filtering or remove it from this method

**Review by romanrudyidev — CHANGES_REQUESTED** [review]

File: src/main/java/com/project/Investment/App/controller/jdbc/RestControllerEntityJdbc.java (hunk 1 above)
Comment (AndriyHishchak) [reply src/main/java/com/project/Investment/App/controller/jdbc/RestControllerEntityJdbc.java]: Done

File: src/main/java/com/project/Investment/App/controller/jdbc/RestControllerEntityJdbc.java [hunk 2]
```diff
@@ -1,43 +1,49 @@
...
 
 import javax.validation.Valid;
 import java.net.URI;
+import java.time.LocalDate;
 import java.util.List;
 import java.util.Optional;
 
 @RestController
-@RequestMapping("/v1/entity/jdbc/")
+@RequestMapping("/entity")
```
Comment (romanrudyidev) [inline src/main/java/com/project/Investment/App/controller/jdbc/RestControllerEntityJdbc.java]: You should use the plural - `/entities`

File: src/main/java/com/project/Investment/App/controller/jdbc/RestControllerEntityJdbc.java [hunk 3]
```diff
@@ -1,43 +1,49 @@
...
-    @GetMapping("{id}")
-    public Entity getById(@PathVariable(name = "id") String id) {
-        return service.findById(id);
+    @GetMapping("/{id}")
+    public ResponseEntity<List<Entity>> getById(
+            @PathVariable(name = "id") String id,
+            @DateTimeFormat(iso = DateTimeFormat.ISO.DATE) @RequestParam(value = "effectiveDate", required = false) Optional<LocalDate> effectiveDate,
+            @RequestParam(value = "limit") Optional<Integer> limit
+    ) {
+        return ResponseEntity.status(HttpStatus.OK).body(service.findById(id, effectiveDate, limit)));
```
Comment (romanrudyidev) [inline src/main/java/com/project/Investment/App/controller/jdbc/RestControllerEntityJdbc.java]: There is `ResponseEntity.ok()` method designed for this purpose

File: src/main/java/com/project/Investment/App/controller/jdbc/RestControllerEntityJdbc.java [hunk 4]
```diff
@@ -49,12 +55,12 @@ public Entity getById(@PathVariable(name = "id") String id) {
...
     @PutMapping("/{id}")
     public ResponseEntity<Entity> update(@PathVariable("id") String id, @Valid @RequestBody EntityDtoRequest entity) {
-        return new ResponseEntity<>(service.update(id, entity), HttpStatus.CREATED);
+        return ResponseEntity.status(HttpStatus.OK).body(service.update(id, entity));
     }
 
     @DeleteMapping("/{id}")
     public ResponseEntity<Entity> deleteById(@PathVariable("id") String id) {
-        return new ResponseEntity<>(service.deleteEntity(id), HttpStatus.ACCEPTED);
+        return ResponseEntity.status(HttpStatus.ACCEPTED).body(service.deleteEntity(id));
```
Comment (romanrudyidev) [inline src/main/java/com/project/Investment/App/controller/jdbc/RestControllerEntityJdbc.java]: Use `ResponseEntity.accepted()` instead

Comment (AndriyHishchak) [reply src/main/java/com/project/Investment/App/controller/jdbc/RestControllerEntityJdbc.java]: Done

File: src/main/java/com/project/Investment/App/controller/jdbc/RestControllerEntityJdbc.java (hunk 3 above)
Comment (AndriyHishchak) [reply src/main/java/com/project/Investment/App/controller/jdbc/RestControllerEntityJdbc.java]: Done

File: src/main/java/com/project/Investment/App/controller/jdbc/RestControllerEntityJdbc.java (hunk 2 above)
Comment (AndriyHishchak) [reply src/main/java/com/project/Investment/App/controller/jdbc/RestControllerEntityJdbc.java]: Done

**Review by romanrudyidev — APPROVED** [review]

**Pull request merged**